docs/UserGuide.md

```diff
@@ -148,6 +152,18 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+### Uploading cards : `upload`
+
+Imports cards to LingoGO! using a CSV file.
+
+Format: `upload CSV_FILE_PATH`
+
+* The file path can be absolute or relative to the location of the LingoGO! file.
+* The CSV file must have 2 columns. The first column is for the front page of the flashcard. The second column is for the other side.
```

Comment:
Perhaps we can include a screenshot to show users an example CSV file so that it is easier for them to understand?